templates/bundles/PrestaShopBundle/Admin/Sell/Catalog/Product/Blocks/tabs.html.twig

```diff
@@ -25,6 +25,7 @@
 {% set isBasicTabValid = productForm.basic.vars.valid %}
 {% set isOptionsTabValid = productForm.options.vars.valid %}
 {% set isPricingTabValid = productForm.pricing.vars.valid %}
+{% set isDescriptionTabValid = productForm.description.vars.valid %}
```

Comment:
Don't worry about splitting the tabs for now it will be handled in a separate PR
You just need to move categories into the BasicType and it will be rendered at the end of the first tab automatically thanks to `form_row(productForm.basic)`

You will just need to apply the categories theme on the right new field:
```twig
{% form_theme productForm.basic.categories '@PrestaShop/Admin/Sell/Catalog/Product/Form/categories_form_theme.html.twig' %}
```

This way you impact less files and there will be less conflicts with the tab splitting PR